synthea-hiv/generator/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile used to create gobbledegook/synthea
+
+FROM openjdk:16-jdk-alpine
+RUN apk --no-cache add openssl wget git curl
+
+WORKDIR /synthea-hiv
+RUN wget https://github.com/synthetichealth/synthea/releases/download/master-branch-latest/synthea-with-dependencies.jar
+COPY hiv_simple.json hiv_simple.json
+COPY hiv_simple hiv_simple
+
+ENV POP 100
+
+CMD java -jar synthea-with-dependencies.jar \
+    -p ${POP} \
+    --exporter.years_of_history 0 \
+    --generate.only_alive_patients true \
+    --generate.max_attempts_to_keep_patient 100 \
+    -d ./ \
+    -m hiv_simple \
+    -k hiv_simple.json
```

Comment:
Can you please add a README for this directory describing how the models here can be used and how they can be modified? For example, how is the generated data persisted on the host machine?

Reply:
The README is part of this PR:
https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/212/files#diff-9b20d8ca70570e2d38dcbd7d1d30a4d8ee2ce340ede936b8794796d532364f4f

I can wait for that PR to be merged and edit the README if needed?